### PR TITLE
Lint and tests

### PR DESCRIPTION
**Pull request by nidemidovich**

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    steps:
+      - name: Setup dependencies
+        uses: ExpressApp/github-actions-poetry@v0.2
+        with:
+          python-version: ${{ matrix.python-version }}
+          poetry-version: "1.1.12"
+
+      - name: Run lint
+        run: |
+          poetry run ./scripts/lint
```
Comment (lxmnk) [inline .github/workflows/actions.yml:38]: No newline

File: scripts/format [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+find pybotx_smart_logger -type f -name "*.py" | xargs add-trailing-comma --py36-plus --exit-zero-even-if-changed
+find tests -type f -name "*.py" | xargs add-trailing-comma --py36-plus --exit-zero-even-if-changed
+
+# This `black` is needed again in order to transfer parameters/arguments to new lines
+# after inserting commas.
+# The first `black` won't be able to transfer parameters/arguments to new lines because
+# there is no comma at the end of the line.
+# Inserting commas must be after the first `black`, so that there is one new line break,
+# if the line is out of max-line-length.
+black pybotx_smart_logger tests > /dev/null
```
Comment (lxmnk) [inline scripts/format:21]: No newline

File: scripts/lint [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+
+black --check --diff pybotx_smart_logger tests
+isort --profile black --check-only pybotx_smart_logger tests
+
+mypy pybotx_smart_logger tests
+flake8 pybotx_smart_logger tests
```
Comment (lxmnk) [inline scripts/lint:9]: No newline

File: scripts/test [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -ex
+
+pytest ${@}
```
Comment (lxmnk) [inline scripts/test:5]: No newline

**Review by lxmnk — APPROVED** [review]

File: pyproject.toml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pybotx-smart-logger"
-version = "0.6.9"
+version = "0.7.9"
```
Comment (Nimond) [inline pyproject.toml:3]: 0.7.0

**Review by Nimond — APPROVED** [review]

**Pull request merged**